pkg/rke/k8s_rke_system_images.go

```diff
@@ -11058,6 +11142,46 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.23.0",
 		},
+		// Enabled in Rancher v2.9.3
+		"v1.30.6-rancher1-1": {
```

Comment:
we'd need to change default in versions file to v1.30.6 too

Reply:
Yes, that's what was causing the RKE PR to fail. Updated